src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Compilation.targets

```diff
@@ -162,6 +162,7 @@ Copyright (c) .NET Foundation. All rights reserved.
          ResponseFiles="$(CompilerResponseFile)"
          RuntimeMetadataVersion="$(RuntimeMetadataVersion)"
          SharedCompilationId="$(SharedCompilationId)"
+         SkipAnalyzers="true"
```

Comment:
@ajaybhargavb @NTaylorMullen @pranavkm is there a downside here (if it solves the CI problems)❔

Reply:
This one's problematic. For instance, the "don't use HtmlHelper.Partial" analyzer that we wrote needs to run here (among other analyzers). We might also need analyzers once we start using source generators for Razor. 

Is there a reason to do this?

Reply:
I introduced this in https://github.com/dotnet/aspnetcore/pull/27653/commits/97bd5d5dd4309edee81c4e0c5b150f5d69c68291 The PublicAPI analyzer was still running in the pre-build for *.cshtml` and `*.razor` files. That meant the analyzer failed the build due to missing `public` methods et cetera. If this is a problem, I could spend time looking for a way to disable just that analyzer and just for this task.